compiler/noirc_frontend/src/elaborator/mod.rs

```diff
@@ -225,7 +233,9 @@ impl<'context> Elaborator<'context> {
             this.collect_impls(typ, *module, impls);
         }
 
-        // resolver resolves non-literal globals here
```

Comment:
Nit: We deleted the comment here but kept "the resolver filters literal globals first" above. Could we bring the comments over from the existing resolver why we have this separation?

Reply:
Good catch, I've removed these comments and copied over the comments from dc_crate instead.